simplic-user/Simplic.User.UI/Views/UserDetailsView.xaml

```diff
@@ -0,0 +1,86 @@
+        </Grid.RowDefinitions>
+
+        <Grid Grid.Column="0" Grid.Row="0" Margin="0,20,0,0">
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width=".3*" />
+                <ColumnDefinition Width="*" />
+            </Grid.ColumnDefinitions>
+            <Grid.RowDefinitions>
+                <RowDefinition Height="Auto" />
+                <RowDefinition Height="Auto" />
+                <RowDefinition Height="Auto" />
+            </Grid.RowDefinitions>
+
+            <Label Grid.Column="0" Grid.Row="0" Content="First Name:" Margin="5,5,0,5" />
+            <TextBox Grid.Column="1" Grid.Row="0" Text="{Binding Path=FirstName}" Margin="0,5,5,5" VerticalContentAlignment="Center" TabIndex="0" />
+
+            <Label Grid.Column="0" Grid.Row="1" Content="User Name:" Margin="5,0,0,5" />
+            <TextBox Grid.Column="1" Grid.Row="1" Text="{Binding Path=UserName}" Margin="0,0,5,5" VerticalContentAlignment="Center" TabIndex="2"
+                     IsEnabled="{Binding Path=IsCreate}"/>
+
+            <Label Grid.Column="0" Grid.Row="2" Content="Phone:" Margin="5,0,0,5" />
+            <TextBox Grid.Column="1" Grid.Row="2" Text="{Binding Path=Phone}" Margin="0,0,5,5" VerticalContentAlignment="Center" TabIndex="4" />
+        </Grid>
+
+        <Grid Grid.Column="1" Grid.Row="0" Margin="0,20,0,0">
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width=".3*" />
+                <ColumnDefinition Width="*" />
+            </Grid.ColumnDefinitions>
+            <Grid.RowDefinitions>
+                <RowDefinition Height="Auto" />
+                <RowDefinition Height="Auto" />
+                <RowDefinition Height="Auto" />
+            </Grid.RowDefinitions>
+
+            <Label Grid.Column="0" Grid.Row="0" Content="Last Name:" Margin="5,5,0,5" />
+            <TextBox Grid.Column="1" Grid.Row="0" Text="{Binding Path=LastName}" Margin="0,5,5,5" VerticalContentAlignment="Center" TabIndex="1" />
+
+            <Label Grid.Column="0" Grid.Row="1" Content="Email:" Margin="5,0,0,5" />
+            <TextBox Grid.Column="1" Grid.Row="1" Text="{Binding Path=Email}" Margin="0,0,5,5" VerticalContentAlignment="Center" TabIndex="3" />
+
+            <Grid Grid.Column="0" Grid.Row="2" Grid.ColumnSpan="2" HorizontalAlignment="Right">
+                <Button Height="27" Width="170" Margin="10,0,5,0" Content="Change password"
+                        Visibility="{Binding Path=IsCreate, Converter={StaticResource FalseVisibilityConverter}}"
+                        Command="{Binding Path=ChangePasswordCommand}"/>
+                <StackPanel Width="170" Margin="10,0,5,0" Orientation="Horizontal"
+                            Visibility="{Binding Path=IsCreate, Converter={StaticResource TrueVisibilityConverter}}">
+                    <Label Content="Is AD user:" />
+                    <CheckBox Width="20" IsChecked="{Binding Path=IsADUser}" Margin="35,0,5,5" VerticalAlignment="Center" />
```

Comment:
We have our own checkbox implementation in simplic:CheckBox. This contains some custom behaviour.

Reply:
Done